### PR TITLE
Shotgun VPC endpoint in all AWS regions

### PR DESCRIPTION
**Pull request by david-richer-adsk**

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/a164e697e66056294cb4981b9f95b86edc2480e4/index.html)

File: docs/en/offerings/spc/setup/traffic_segregation.md [hunk 1]
```diff
@@ -9,26 +9,24 @@ lang: en
...
 
-* For the security group, Shotgun only requires the inbound port tcp/443 to be open
+You need to configre your office DNS server to resolve your shotgun site to your Shotgun VPC Endpoint DNS name.
 
-* Please contact Shotgun Support to get the endpoint approved, so that you can start using it
+Example DNS entry
 
-![Create endpoint](../images/spc-endpoint-create_privatelink.png)
+```
+mystudio-staging.shotgunstudion.com   CNAME   vpce-048447456a4f57e14-1j3wh50q.vpce-svc-0b054415458f57634.us-west-2.vpce.amazonaws.com
```
Comment (comeaug) [inline docs/en/offerings/spc/setup/traffic_segregation.md]: mystudio-staging.shotgunstudion.com should be mystudio-staging.shotgunstudio.com

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/d9324da1686c2c087a65d17882791093c6273100/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/3b91d27d1e870fb494c8622d93f2147d983c7a1d/index.html)

**Pull request merged**